qmetaobject_impl/src/lib.rs

```diff
@@ -26,11 +27,13 @@ CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 extern crate syn;
 #[macro_use]
 extern crate quote;
-
+extern crate darling;
```

Comment:
I think we no longer need extern crate.

Reply:
True. Just tried to be consistent there.